### ISSUE TITLE
Theme not applied on startup on Chrome, Cannot GET /theme/css/theme.min.css

### ISSUE THREAD

**Issue by colinl** [node-red-contrib-themes/theme-collection#70]
# Problem/Motivation

- Theme version: midnight red 2.2.0 but happens on other themes too
- Node-RED version: 2.2.2
- Platform/OS: Ubuntu 20.04
- Browser: Chrome 99.0.4844.51

## Steps to reproduce

Close the browser (Chrome) with the editor open in a tab
Restart node-red
Open the browser

The editor appears but the theme is not applied, the default theme appears to be used.
The browser console shows
```
Refused to apply style from 'http://localhost:1880/theme/css/theme.min.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
:1880/#flow/bdd7be38.d3b55:1 Refused to apply style from 'http://localhost:1880/theme/css/theme-customizations.min.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

To get the theme applied refresh the browser page.
Once the page has been refreshed then there is no further problem.  The browser may be closed and opened and the theme continues to be used.  To see the problem node-red must be restarted and the browser opened.

I believe the problem is that if node-red is restarted and then `http://localhost:1880/theme/css/theme.min.css` is entered into the browser window the response is
`Cannot GET /theme/css/theme.min.css`
If the node-red editor page is then refreshed then that url shows the theme as expected.

The problem is not seen with Firefox so I guess it is a timing issue to do with when the theme css becomes available.
Perhaps this should be reported against node-red, I don't know.

See also this post on the forum https://discourse.nodered.org/t/themes-from-themes-collection-not-applied-on-startup-on-chrome/59383

**Comment by knolleary** [comment]
> Perhaps this should be reported against node-red, I don't know.

Yes it should. The theme isn't responsible for how the files get served up - that is the core project's doing. The theme just provides the files to the runtime so it can serve them.
